src/v2/Apps/Consign/Routes/MarketingLanding/Components/GetPriceEstimate/ArtistInsightResult.tsx

```diff
@@ -44,22 +62,21 @@ export const ArtistInsightResult: React.FC = () => {
           <Box pr={4} mb={[2, 2, 0]}>
             <Image
               src={imageUrl}
-              width={[80, 80, 120]}
-              height={[80, 80, 120]}
+              width={[80, 80, 122]}
+              height={[80, 80, 122]}
             />
           </Box>
           <Box>
-            <Text
-              variant="largeTitle"
-              borderBottom={`1px solid ${color("black60")}`}
-              pb={0.5}
-              mb={2}
-            >
+            <Text variant="largeTitle" mt="-6px">
```

Comment:
The above changes were to fix some of the alignments, to make things more exact: 

<img width="147" alt="Screen Shot 2020-12-01 at 11 38 45 PM" src="https://user-images.githubusercontent.com/236943/100842966-7de17680-342e-11eb-8f82-7496cf37b20b.png">